components/templates/BasePage.tsx

```diff
@@ -33,66 +33,102 @@ type Props = {
-            </div>
-            <div style={{ float: "right", width: "3%", marginLeft:"10%"}}>
-              <IconButton onClick={() => router.push(`/${seoName}/login-sponsor`)}>
-                <PersonIcon style={{color:"#6c6c6c"}} />
+                  <Link
+                    underline="none"
+                    href={`/${seoName}`}
+                    className={classes.titleLight}
+                  >
+                    {title}
+                  </Link>
+                </Typography>
+              </div>
+              <div style={{ float: "left", width: "10%"}}>
+                <Typography>
+                  <Link
+                    underline="none"
+                    href={`/${seoName}/about`}
+                    className={classes.titleLight}
+                  >
+                    About Us
+                  </Link>
+                </Typography>
+              </div>
+              <div style={{ float: "left", width: "10%"}}>
+                <Typography>
+                  <Link
+                    underline="none"
+                    href={`/${seoName}/help`}
+                    className={classes.titleLight}
+                  >
+                    Get involved
+                  </Link>
+                </Typography>
+              </div>
+              <div style={{ float: "left", backgroundColor: `${MuiTheme.palette.primary.main}`, marginLeft:"10%", width:"100px" }}>
+                <Button size="large" style={{color:'white', paddingLeft:"20%"}}  href={`/${seoName}/payment`}>Donate</Button>
+              </div>
+              <div style={{ float: "right", width: "3%", marginLeft:"10%"}}>
+                <IconButton onClick={() => router.push(`/${seoName}/login-sponsor`)}>
+                  <PersonIcon style={{color:"#6c6c6c"}} />
+                </IconButton>
+              </div>
+          </div>
+          ) : (
+            <div style={{marginTop:"18px"}}>
+              <IconButton edge="start" aria-haspopup="true" onClick={handleMenuClick}>
+                <MenuIcon />
               </IconButton>
-            </div>
+              <Menu
+                anchorEl={anchorEl}
+                keepMounted
+                open={Boolean(anchorEl)}
+                onClose={handleMenuClose}
+              >
+                <MenuItem onClick={() => router.push(`/${seoName}/about`)}>About Us</MenuItem>
+                <MenuItem onClick={() => router.push(`/${seoName}/help`)}>Get involved</MenuItem>
+                <MenuItem onClick={() => router.push(`/${seoName}/login-sponsor`)}>My Account</MenuItem>
+              </Menu>
+              <div style={{ float: "right", backgroundColor: `${MuiTheme.palette.primary.main}`, width:"100px", marginLeft:"300px"}}>
+                <Button size="medium" style={{color:'white', paddingLeft:"20%"}}  href={`/${seoName}/payment`}>Donate</Button>
+              </div>
```

Comment:
This section is really interesting

Reply:
that's this top nav bar with the menu, does it make sense?
<img width="357" alt="Screenshot 2021-05-04 at 09 14 46" src="https://user-images.githubusercontent.com/44492659/116976516-2daa6100-acb9-11eb-8439-df8f08c1f12d.png">